src/app/businesscard/(form)/1/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { GenerateItem, GenerateView } from '@/components';
+import { button, input } from '@/styles/ogoo';
+import { flexCol, flexRow } from '@/styles/ogoo/alignment.css';
+import { subText } from '@/styles/ogoo/colors.css';
+import { cn } from '@/utils';
+
+export default async function BusinesscardOne() {
+  return (
+    <GenerateView
+      questionNumber={'Q1'}
+      firstTitle={'집사님의 반려동물에 대해'}
+      secondTitle={'알려주세요'}
+      nextLink={'/businesscard/2'}
+    >
```

Comment:
```suggestion
    <GenerateView
      questionNumber={1}
      title={'집사님의 반려동물에 대해\n알려주세요'}
      nextLink={'/businesscard/2'}
    >
```

이런 식으로 수정해보는 건 어떨까요??